storage/gcloud/aio/storage/storage.py

```diff
@@ -57,6 +57,59 @@ def __init__(self, *, service_file: Optional[Union[str, io.IOBase]] = None,
     def get_bucket(self, bucket_name: str) -> Bucket:
         return Bucket(self, bucket_name)
 
+    async def copy(self, bucket: str, object_name: str,
+                   destination_bucket: str, *, new_name: str = None,
```

Comment:
optional nit: I find it's easier to parse function arguments when the default and keyword arguments are separated, so I'd move `*, new_name: str = None,` to the next line

Reply:
I disagree here -- I think @jonathan-johnston did the right thing in sticking with the same coding conventions used throughout this repo. If we want to change convention like that, we should do it everywhere so that each chunk of this library doesn't have a different style. Lets leave this as-is for now and create a follow-up PR touching everything if we think that's worth it.